hermes/probe/delete/delete_test.go

```diff
@@ -101,7 +101,7 @@ func genTestTarget(cfg *monitorpb.HermesProbeDef, t *testing.T) *probe.Target {
 
 	metrics, err := m.NewMetrics(cfg, genTargetPb())
 	if err != nil {
-		t.Fatalf("could not initialise metrics using config and target provided, err: %v", err)
+		t.Fatalf("could not initialise metrics using config and target provided, %v", err)
```

Comment:
Nit: Use ": " as the separator instead of ", ".

Reply:
Ack.